rtl/__init__.py

```diff
@@ -279,37 +280,60 @@ def vhdlentityfiles(self,value):
-
-    #    The content of the file is defined in interactive_control.
-    #    '''
-    #    if not hasattr(self, '_interactive_control_file'):
-    #        obsoletepath = '%s/Simulations/rtlsim/dofile.do' % (self.entitypath)
-    #        if os.path.exists(obsoletepath):
-    #            self.print_log(type='O',msg='Found deprecated do-file %s' % obsoletepath)
-    #            self.print_log(type='O',msg='Define the desired contents of the do-file in interactive_control -property in the testbench and delete the deprecated file.')
-    #            self.print_log(type='O',msg='Loading the deprecated file for now.')
-    #            self._interactive_control_file = obsoletepath
-    #        else:
-    #            self._interactive_control_file = '%s/dofile.do' % self.simpath
-    #    return self._interactive_control_file
-    #@interactive_control_file.setter
-    #def interactive_control_file(self,value): 
-    #    self._interactive_control_file = value
+    @property
+    def interactive_control_contents(self):
+        ''' Content of the interactive rtl control file (.do -file).
+
+        If this property is set, a new dofile gets written to the simulation
+        path.
+
+        For example, the contents can be defined in the top testbench as::
+
+            self.interactive_control_contents="""
+                add wave -position insertpoint \\
+                sim/:tb_inverter:A \\
+                sim/:tb_inverter:clock \\
+                sim/:tb_inverter:Z
+                run -all
+                wave zoom full
+            """
+        
+        '''
+        if not hasattr(self, '_interactive_control_contents'):
+            self._interactive_control_contents = ''
+        return self._interactive_control_contents
+    @interactive_control_contents.setter
+    def interactive_control_contents(self,value): 
+        self._interactive_control_contents = value
+
+    @property
+    def interactive_controlfile(self):
+        ''' Path to interactive rtl control file (.do -file).
+
+        The content of the file can be defined in interactive_control. If the
+        content is not set in interactive_control property, the do-file is read
+        from this file path.
+        '''
+        if not hasattr(self, '_interactive_controlfile'):
+            obsoletepath = '%s/Simulations/rtlsim/dofile.do' % self.entitypath
+            dofilepath = '%s/dofile.do' % self.simpath
+            if os.path.exists(obsoletepath):
+                self.print_log(type='O',msg='Found deprecated do-file in %s' % obsoletepath)
+                if os.path.isfile(dofilepath):
+                    self.print_log(type='O',msg='File exists also in %s' % dofilepath)
+                else:
+                    self.print_log(type='O',msg='Copying the file to %s' % dofilepath)
+                    shutil.copyfile(obsoletepath,dofilepath)
+                self.print_log(type='O',msg='Please remove the deprecated file.')
```

Comment:
This message should also instruct what parameters to use to configure the dofile location/contents, for example:
```suggestion
                self.print_log(type='O',msg='Please remove the deprecated file and use the `interactive_controlfile` attribute instead.')
```